dev/merge_spark_pr.py

```diff
@@ -142,6 +142,9 @@ def merge_pr(pr_num, target_ref, title, body, pr_repo_desc):
         distinct_authors[0])
     if primary_author == "":
         primary_author = distinct_authors[0]
+    else:
+        # When primary author is specified manually, put it at the head of author list.
+        distinct_authors.insert(0, primary_author)
```

Comment:
@viirya good catch. When we put it at the head of the author list, do we need to dedupe it again?

Reply:
I gave it a thought, but didn't do it. If you think it is necessary, I will push another commit for it.